crates/bevy_picking/src/events.rs

```diff
@@ -285,6 +286,19 @@ pub struct DragEntry {
     pub latest_pos: Vec2,
 }
 
+/// Fires while a pointer is scrolling over the `target` entity.
```

Comment:
```suggestion
/// Fires while a pointer is scrolling over the `target` entity.
```

The way this is written seems a bit odd, does a pointer scroll? Maybe I'm being fussy, I'm not sure.

Reply:
This is just mirroring the language used in the other event docs. I think it's a valid description. The pointing device sends the scroll, but it is definitely tied to the pointer. That is, pointer scrolling events depend on where the pointer is located w.r.t. picking, it's not just an input.

Reply:
Does that mean we should leave it as is?

Reply:
Yeah it's fine, I was just being nitpicky.